File: python_programming_basics/while_loops/l_02_password.py
username = input()
correct_password = input()
new_password = input()

while new_password != correct_password:
    new_password = input()

print(f"Welcome {username}!")

# solution 2
# username = input()
# correct_password = input()
#
# while True:
#     password = input()
#     if password != correct_password:
#         continue
#
#     else:
#         print(f"Welcome {username}!")
#         break

File: JavaScript Basics/05. While Loop/l-02-Password.js
function login(input) {
    let username = input[0];
    let password = input[1];

    let data = input[2];
    let counter = 3;

    while (data !== password) {
        data = input[counter];
        counter++;
    }

    console.log(`Welcome ${username}!`);
}